src/pages/Components/Radio/specs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Specs'
+order: 3
+---
+
+## Properties
+
+### CSS Tokens : layout
+
+| State       | Property          | Token                |
+| ----------- | ----------------- | -------------------- |
+| **Default** | border-radius     | `50%`                |
+| **Default** | border-width      | `border.m` (2px)     |
+| **Default** | min-width         | `20px`               |
+| **Default** | width             | `$mu125`             |
+| **Default** | min-height        | `20px`               |
+| **Default** | height            | `$mu125`             |
+
+### CSS Tokens : state colors
+
+> **You must always use the tokens, the colors are only there for that information.**
+
+| State                   | Property     | Token                              | Color   |
+| ----------------------- | ------------ | ---------------------------------- | ------- |
+| **Default**             | background   | `color.input.background`           | #ffffff |
```

Comment:
should we add the base token `color.primary...` or something like `color.primary... (#ffffff)` ?

Reply:
Not sure to understand you